examples/base_difference_mapcubes.py

```diff
@@ -0,0 +1,37 @@
+"""
+========================================
+Base Difference in Mapcubes
+========================================
+
+This example illustrates the working of base difference using Mapcubes. We use the AIA_193 series and using a base map we calculate
+the base difference of all the maps in the mapcube with respect to the base map.
+"""
+############################################################################
+
+# Importing Libraries
+import matplotlib.pyplot as plt
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+# Forming the mapcube consisting of maps formed by the AIA_193 series fits files
+aiamap = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE, cube=True)
```

Comment:
I wonder  if renaming this to make it clearer it's a mapcube and not just a map.